### DISCUSSION TITLE
milvus 如何集成prometheus?

### DISCUSSION THREAD

**Discussion by Anandonzy** [milvus-io/milvus#18109] [Q&A and General discussion]
Hello 我参见官方文档集成milvus,目前已经给模板json导入了,然后发现无法使用.
感觉应该是需要修改配置文件,但是文档没看到相关的内容.
<img width="762" alt="image" src="https://user-images.githubusercontent.com/32696359/177504217-83bed38f-3bc3-4c65-a742-4d62c4748486.png">
<img width="1360" alt="image" src="https://user-images.githubusercontent.com/32696359/177504233-af505fae-1b60-469b-88b7-6330098dd1fa.png">

**Comment by Anandonzy** [comment]
hello

**Comment by xiaofan-luan** [comment]
@LoveEachDay pls help on this~

**Comment by LoveEachDay** [comment]
@Anandonzy Could you provide grafana version and did you deploy grafana in the kubernetes cluster?

**Reply by Anandonzy** [reply]
Grfana5.1.1 
Not in k8s.

**Reply by LoveEachDay** [reply]
> Grfana5.1.1 Not in k8s.

You'd better use Grafana of version 8+, and deploy in Kubernetes because our dashboard has some dependencies on Kubernetes metrics. Or you can create your own dashboard and take the metrics references from https://github.com/milvus-io/milvus/blob/master/deployments/monitor/grafana/README.md.
